Docs/source/Software/Apps.rst

```diff
@@ -176,11 +176,14 @@ Voltage
 ========
 
 Purpose
-    This App is used to keep track of the voltages of all the modules in the battery pack.
+    This App is used to keep track of the voltages and open wires of all the modules in the 
+    battery pack.
 
 Usage
-    ``Voltage_Init()`` must be called before calling any other Voltage functions. Descriptions of
-    other functions are located in Voltage.h.
+    ``Voltage_Init()`` must be called before calling any other Voltage functions. All of the data
+    used by the functions is based on data collected by the function ``Voltage_UpdateMeasurements()``
+    so it might be necessary to call ``Voltage_UpdateMeasurements()`` before any other function.
```

Comment:
The functions that care about voltages use `Voltage_UpdateMeasurements()`. The open wire functions all directly contact the LTC. Also, I realized when reading through the code again that `Voltage_OpenWireSummary()` requires UART to be initialized, since it uses `printf()`.